casual/casual-caller/src/main/java/se/laz/casual/connection/caller/jmx/CasualCallerControl.java

```diff
@@ -6,57 +6,107 @@
 
 package se.laz.casual.connection.caller.jmx;
 
+import se.laz.casual.api.queue.QueueInfo;
 import se.laz.casual.connection.caller.Cache;
 import se.laz.casual.connection.caller.ConnectionFactoryEntry;
+import se.laz.casual.connection.caller.ConnectionFactoryEntryStore;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
 import java.util.stream.Collectors;
 
 public class CasualCallerControl implements CasualCallerControlMBean
 {
     private final Cache cache;
+    private final ConnectionFactoryEntryStore connectionFactoryEntryStore;
 
-    public CasualCallerControl(Cache cache)
+    public CasualCallerControl(Cache cache, ConnectionFactoryEntryStore connectionFactoryEntryStore)
     {
         this.cache = cache;
+        this.connectionFactoryEntryStore = connectionFactoryEntryStore;
     }
 
     @Override
-    public void purgeServiceCache()
+    public List<String> poolsValid()
```

Comment:
You have changed the ordering of the words for some of the methods which for me makes them not nice to read.
action / adjective then noun is much nicer to read:
validPools rather than poolsValid
invalidPools rather than poolsInvalid
purgeServiceCache rather than serviceCachePurge
etc

Reply:
For this one I was on the fence but finally elected to do this way to have the methods sort by category. After using these myself i have to agree these are not fun to read at all. I'll go change it back to names that make sense to read